docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -97,20 +99,34 @@ which should produce output like this:
 
 The IP address is listed next to `LoadBalancer Ingress`.
 
-## Annotation to modify the LoadBalancer behavior for preservation of Source IP
-In 1.5, a Beta feature has been added that changes the behavior of the external LoadBalancer feature.
+## Modify the LoadBalancer behavior for preservation of Source IP
+
+Due to the implementation of this feature, the source IP for sessions as seen in the target
+container will *not be the original source IP* of the client. This is the default behavior as
+of Kubernetes v1.7. However, starting in v1.5, an optional beta feature has been added that
+will preserve the client Source IP for GCE/GKE environments. This feature has been promoted
+to GA in v1.7, with below APIs defined:
 
-This feature can be activated by adding the beta annotation below to the metadata section of the Service Configuration file.
+* `Service.Spec.ExternalTrafficPolicy` - It denotes if this Service desires to route
```

Comment:
Suggested change: `service.spec.externalTrafficPolicy`

Specify fields as they appear in YAML/JSON manifests, not in the API: